versions/challengeutils.txt

```diff
@@ -0,0 +1 @@
+sagebionetworks/challengeutils:v4.2.0
```

Comment:
TIL: This is so neat!  This way, we only need to edit these text files for all CWL tools!